client/gui-qt/minimap.cpp

```diff
@@ -113,177 +113,54 @@ void minimap_view::draw_viewport(QPainter *painter)
     return;
```

Comment:
The variables `i`, `src_x`, `src_y`, `dst_x`, and `dst_y` (declared above) are no longer used.